compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -5,10 +5,9 @@ package diagnostic
 
 import dotc.core._
 import Contexts.Context, Decorators._, Symbols._, Names._, NameOps._, Types._
-import util.{SourceFile, NoSource}
-import util.{SourcePosition, NoSourcePosition}
+import util.SourcePosition
```

Comment:
These imports that were removed were unused, that's why I removed them. `util.SourcePosition` still needs to be included as it is in use in the MessageContainers.

I can put the imports back in if that's what you want